src/libs/Visibility/index.desktop.js

```diff
@@ -0,0 +1,38 @@
+import ELECTRON_EVENTS from '../../../desktop/ELECTRON_EVENTS';
+
+/**
+ * Detects whether the app is visible or not. Electron supports
+ * document.visibilityState, but switching to another app while
+ * Electron is partially occluded will not trigger a state of hidden
+ * so we ask the main process synchronously whether the
+ * BrowserWindow.isFocused()
```

Comment:
NAB, these lines look a little short. We should be able to make them a bit longer.